arangod/Aql/ExecutionPlan.cpp

```diff
@@ -737,7 +738,17 @@ ModificationOptions ExecutionPlan::parseModificationOptions(AstNode const* node)
         } else if (name == "exclusive") {
           options.exclusive = value->isTrue();
         } else if (name == "overwrite") {
-          options.overwrite = value->isTrue();
+          if(value->isTrue()) {
+            options.overwrite = true;
+          }
+        } else if (name == "overwriteMode") {
+          auto ref = value->getStringRef();
```

Comment:
this is a bit dangerous, as the `getStringRef()` assumes the value is actually a string.
If it is not a string, it will throw a VelocyPack exception here.